index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Border Animation</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="frem">
+        <div class="rotate"></div>
+        <div class="cant">
+            <div class="area">
+                <img src="./image/discord.png" alt="">
```

Comment:
Why alt attribute  is empty describe about image into alt attribute

Reply:
oops i was forget to add .. just updated it. thank you